src/ast/mod.rs

```diff
@@ -857,6 +857,8 @@ pub enum Statement {
         default_charset: Option<String>,
         collation: Option<String>,
         on_commit: Option<OnCommit>,
+        clickhouse_order: Option<Vec<Ident>>,
```

Comment:
Can you please change this field name to `order by` to match the SQL more closely (and perhaps add a comment that it is to support the clickhouse variant)?